Tensile/Source/SolutionMapper.h

```diff
@@ -22,8 +22,16 @@
 
 #pragma once
 
+#if defined(max)
+#undef max
+#endif
 #include <cstddef>
 #include <limits>
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
If we can get away with it, can we try adding the #define WIN32_LEAN_AND_MEAN here ? 

Alternatively, I might tuck this env variable code into Utils.h/.cpp and drag the windows headers into the compilation unit only.